src/sentry/static/sentry/app/components/events/interfaces/rawExceptionContent.jsx

```diff
@@ -96,7 +96,7 @@ class RawExceptionContent extends React.Component {
           content = <LoadingIndicator />;
         } else if (this.state.error) {
           content = <LoadingError onRetry={this.fetchData} />;
-        } else if (!this.state.loading && this.state.crashReport != '') {
```

Comment:
Is it possible we (unintentionally) relied on loose equality here?

Reply:
I think this one is ok since it's initialized as empty string in constructor and again in the fetch function